buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1358,6 +1363,14 @@ def set_build_extra_options(build_specs=null) {
         }
 
         EXTRA_MAKE_OPTIONS = params.EXTRA_MAKE_OPTIONS
+        if (params.CODE_COVERAGE) {
+            def CODE_COVERAGE_MAKE_OPTIONS = "EXTRA_CMAKE_ARGS=-DCODE_COVERAGE=ON"
+            if (!EXTRA_MAKE_OPTIONS) {
+                EXTRA_MAKE_OPTIONS = CODE_COVERAGE_MAKE_OPTIONS
+            } else {
+                EXTRA_MAKE_OPTIONS += " " + CODE_COVERAGE_MAKE_OPTIONS
+            }
```

Comment:
I don't think this is right when `EXTRA_MAKE_OPTIONS` already has a value.
Please invert the `if` statement so it doesn't involve negation.

Reply:
The `if` statement has been inverted as suggested.

Reply:
Sorry, I confused `EXTRA_MAKE_ARGS` and `EXTRA_CMAKE_ARGS`.

What I was concerned about was the situation where someone wants to specify `EXTRA_CMAKE_ARGS` _and_ `CODE_COVERAGE`: How will that work?

Reply:
In your aarch64 build, I see
```
12:20:52  + make EXTRA_CMAKE_ARGS=-DCODE_COVERAGE=ON EXTRA_CMAKE_ARGS=-DCODE_COVERAGE=ON all
```
Only one definition of `EXTRA_CMAKE_ARGS` can appear; others will be ignored and have no effect. In this case, they're all the same, but what if they need to be different (and thus combined)?